content/_data/changelogs/lts.yml

```diff
@@ -2837,6 +2837,143 @@
         These were made to anticipate a bugfix in Firefox which has been backed out since.
         (regression in 2.164.3)
 
+- version: 2.204.1
+  date: 2019-12-18
+  lts_predecessor: 2.190.3
+  lts_baseline: '2.204'
+  changes: # compared to lts_baseline
+    - type: rfe
+      pull: 4349
+      authors:
+        - fqueiruga
+      issue: 59508
+      message: |-
+        Show a tooltip with the full link name when hovering over sidebar links.
+    - type: bug
+      pull: 4346
+      authors:
+        - MRamonLeon
+      issue: 59793
+      message: |-
+        Prevent faulty subtask contributors from leaving builds running forever.
+    - type: bug
+      pull: 4298
+      authors:
+        - DarphBoBo
+      issue: 59665
+      message: |-
+        Fix <code>Uninstall</code> column sorting in the Plugin Manager Install pane.
+  lts_changes: # compared to lts_predecessor
+    - type: major rfe
+      pull: 3991
+      authors:
+        - daniel-beck
+      message: |-
+        Redesign password fields to prevent password auto-fill except for the login form.
+        Reduce browsers offering to update stored passwords.
+        Revert by setting the system property <code>hudson.Functions.hidingPasswordFields</code> to <code>false</code>.
+    - type: major rfe
+      pull: 4171
+      issue: 58993
+      authors:
+        - jtnord
+      message: |-
+        Prevent calls to <code>Jenkins#save</code> persisting data before we have finished loading the in-memory model.
+        This prevents possible corruption of the main Jenkins configuration.
+    - type: major rfe
+      issue: 19887
+      message: |-
+        Allow time zone to be set on a per-user basis.
+      pull: 4113
+      authors:
+        - silent-snowman
+    - type: major rfe
+      pull: 4239
+      issue: 41891
+      authors:
+        - daniel-beck
+      message: |-
+        Add an option for a Resource Root URL through which Jenkins will serve user-generated static resources like workspace files or archived artifacts without the need for Content-Security-Policy headers.
+    - type: major rfe
+      pull: 3970
+      authors:
+        - daniel-beck
+      message: |-
+        Remove the ability to download update center metadata using the user's browser (deprecated since 2015).
+        Jenkins will no longer inform about available updates without a connection to update sites.
+        We recommend the use of a local mirror of our update sites, or a self-hosted update center like <a href="https://github.com/jenkinsci/juseppe">Juseppe</a> in these situations.
+    - type: rfe
+      pull: 4242
+      authors:
+        - daniel-beck
+      message: |-
+        Stop bundling Maven Plugin and Subversion Plugin with Jenkins.
```

Comment:
Also https://github.com/jenkinsci/jenkins/pull/4040 ? Can be done in the same upgrade guidelines entry